### ISSUE TITLE
Non invia nessuna notifica a coloro che spuntano la casellina

### ISSUE THREAD

**Issue by gigioooo** [stcr/subscribe-to-comments-reloaded#368]
salve, non capisco perchè ma facendo le prove non invia nessuna notifica email . ho attivato il plugin , h provato a cambiare la mia email come ho letto in un altro post qui. ma nulla.. non succede nulla.

**Comment by Reedyseth** [comment]
Hello @gigioooo the problem might be that you are server are sending the email as anonymouse and for that reason you email notification is rejected. Are you using a commercial email like gmail or hotmail or the one in your domain?

**Comment by gigioooo** [comment]
I tried with the one in My domain, I tried with gmail, and I tried with
Gmail SMTP. Nothing works...

Il 15 ago 2017 01:58, "Israel Barragan C." <notifications@github.com> ha
scritto:

Hello @gigioooo <https://github.com/gigioooo> the problem might be that you
are server are sending the email as anonymouse and for that reason you
email notification is rejected. Are you using a commercial email like gmail
or hotmail or the one in your domain?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/stcr/subscribe-to-comments-reloaded/issues/368#issuecomment-322341446>,
or mute the thread
